bin/slackrunner.rb

```diff
@@ -4,14 +4,22 @@
 require "werewolf"
 
 if ENV['SLACK_API_TOKEN'].nil?
-  puts "ERROR:  Please set the'SLACK_API_TOKEN' environment variable and try again"
+  puts "ERROR:  Please set the 'SLACK_API_TOKEN' environment variable and try again"
   exit(1)
 end
 
 SlackRubyBot::Client.logger.level = Logger::INFO
 game = Werewolf::Game.instance
 
 slackbot = Werewolf::SlackBot.new(token: ENV['SLACK_API_TOKEN'], aliases: ['!', 'w'])
+
+# slackbot.channel = 'C2EP92WF3' # werewolf
+# slackbot.channel = 'G2FQMNAF8' # werewolf-bot-dev
+
+unless ENV['SLACK_CHANNEL'].nil?
```

Comment:
Did it this way so that you could still hardcode it to something and override with environment variable. Should have uncommented one of the above lines, probably dev